docs/book/how-to/popular-integrations/aws-guide.md

```diff
@@ -286,6 +286,90 @@ Now that you have a functional AWS stack set up with ZenML, you can explore more
 * Explore ZenML's [integrations](../../component-guide/README.md) with other popular tools and frameworks in the machine learning ecosystem.
 * Join the [ZenML community](https://zenml.io/slack) to connect with other users, ask questions, and get support.
 
-By leveraging the power of AWS and ZenML, you can streamline your machine learning workflows, improve collaboration, and deploy production-ready pipelines with ease. Happy experimenting and building!
+By leveraging the power of AWS and ZenML, you can streamline your machine
+learning workflows, improve collaboration, and deploy production-ready pipelines
+with ease. What follows is a set of best practices for using your AWS stack with ZenML.
+
+## Best Practices for Using an AWS Stack with ZenML
+
+When working with an AWS stack in ZenML, consider the following best practices
+to optimize your workflow, enhance security, and improve cost-efficiency. These
+are all things you might want to do or amend in your own setup once you have
+tried running some pipelines on your AWS stack.
+
+### Use IAM Roles and Least Privilege Principle
+
+Always adhere to the principle of least privilege when setting up IAM roles. Only grant the minimum permissions necessary for your ZenML pipelines to function. Regularly review and audit your IAM roles to ensure they remain appropriate and secure.
+
+### Leverage AWS Resource Tagging
+
+Implement a consistent tagging strategy for your AWS resources. To tag an S3 bucket:
+
+```shell
+aws s3api put-bucket-tagging --bucket your-bucket-name --tagging 'TagSet=[{Key=Project,Value=ZenML},{Key=Environment,Value=Production}]'
+```
+
+These tags will help you with billing and cost allocation tracking and also with
+any cleanup efforts.
+
+### Implement Cost Management Strategies
+
+Use AWS Cost Explorer and AWS Budgets to monitor and manage your spending. To create a cost budget:
+
+1. Create a JSON file (e.g., `budget-config.json`) defining the budget:
+
+```json
+{
+  "BudgetLimit": {
+    "Amount": "100",
+    "Unit": "USD"
+  },
+  "BudgetName": "ZenML Monthly Budget",
+  "BudgetType": "COST",
+  "CostFilters": {
+    "TagKeyValue": [
+      "user:Project$ZenML"
+    ]
+  },
+  "CostTypes": {
+    "IncludeTax": true,
+    "IncludeSubscription": true,
+    "UseBlended": false
+  },
+  "TimeUnit": "MONTHLY"
+}
+```
+
+2. Create the cost budget:
+
+```shell
+aws budgets create-budget --account-id your-account-id --budget file://budget-config.json
+```
+
+Set up cost allocation tags to track expenses related to your ZenML projects:
+
+```shell
+aws ce create-cost-category --name ZenML-Projects --rules-version 1 --rules file://rules.json
```

Comment:
did you check if this is correct? i think it should be [create-cost-category-definition](https://docs.aws.amazon.com/cli/latest/reference/ce/create-cost-category-definition.html)

Reply:
you're right!